tracex_project/db_results/templates/db_results_overview.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Database Results</title>
+        {% load static %}
+        <link rel="stylesheet" type="text/css" href="{% static '/tracex/css/style.css' %}">
+    </head>
+<body class="main_body">
+    <img src="{% static '/tracex/img/database_results_logo.png' %}" alt="Database Results Logo" class="database_results_logo">
```

Comment:
The logo seems unproportional big 
IDK why, because it should be okay this way, but if I set class="extraction_logo" everything seems normal...
![grafik](https://github.com/bptlab/TracEX/assets/116206992/7f258dfe-8976-4108-a01d-1c3e3118b002)

Reply:
your browser is using the style.css from ur cache thats why the design is off. I will resolve this thread for now, please reopen if u need assistance